net-misc/ptpd/ptpd-2.3.1-r2.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools flag-o-matic systemd
+
+DESCRIPTION="Precision Time Protocol daemon"
+HOMEPAGE="https://github.com/ptpd/ptpd"
+
+SRC_URI="https://github.com/ptpd/ptpd/archive/${P}.tar.gz"
+KEYWORDS="~amd64 ~arm ~x86"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="debug experimental ntp +pcap snmp slave-only +statistics"
+RDEPEND="
+	pcap? ( net-libs/libpcap )
+	snmp? ( net-analyzer/net-snmp )"
+DEPEND="${RDEPEND}"
+RDEPEND="${RDEPEND}
+	ntp? ( net-misc/ntp )"
+
+S=${WORKDIR}/ptpd-${P}
+
+src_prepare() {
+	eapply_user
+	sed -i -e 's/U64/struct counter64/' src/dep/snmp.c || die "sed failed"
+	sed -i -e 's/EVP_MD_CTX/PTPD_EVP_MD_CTX/' src/dep/ntpengine/ntp_isc_md5.h || die "sed failed"
+	sed -i -e 's/EVP_MD_CTX/PTPD_EVP_MD_CTX/' src/dep/ntpengine/ntp_isc_md5.c || die "sed failed"
+	eautoreconf
+}
+
+src_configure() {
+	append-flags -fno-strict-aliasing
+	econf \
```

Comment:
While at it, could you switch this ebuild to use `local myconf=( ... )` array per the modern coding style instead of the backslash hell?